### PR TITLE
Basic templates markup and styles

### LINKED ISSUE #26: 404 page template markup and style

### PR DESCRIPTION
**Pull request by rslnk**
Closes #26, #29,  #50, #54

### CODE REVIEW

File: web/app/themes/theme/404.php [hunk 1]
```diff
@@ -1,7 +1,31 @@
...
+<main class="o-wrapper--row c-404" style="background-image: url(<? echo get_template_directory_uri() ?>/assets/images/homepage-intro-image--1.0.jpg);">
 
-<div class="alert alert-warning">
-  <?php _e('Sorry, but the page you were trying to view does not exist.', 'sage'); ?>
-</div>
+  <div class="u-centering-wrapper js-main-block">
+    <div class="u-centered">
+      <div class="o-wrapper">
+        <div class="c-404__content">
+          <div class="c-404-title">
```
Comment (odincov) [inline web/app/themes/theme/404.php]: Shouldn't be double underscore here?

File: web/app/themes/theme/assets/scripts/main.js [hunk 2]
```diff
@@ -59,6 +59,22 @@
       init: function() {
         // JavaScript to be fired on the about us page
       }
+    },
+    // 404 page
+    '404': {
+      init: function() {
```
Comment (odincov) [inline web/app/themes/theme/assets/scripts/main.js]: Function isn't used.

File: web/app/themes/theme/assets/styles/components/404.styl [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  background-repeat no-repeat
+  background-position 70% 100%
+  background-size: cover
+  +media-query($screen, $bp--tablet--portrait)
+    background-position 50% 30%
+
+.c-404__content
+  col(12)
+
+.c-404-title
```
Comment (odincov) [inline web/app/themes/theme/assets/styles/components/404.styl]: double underscore?

File: web/app/themes/theme/assets/styles/components/404.styl [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  margin rem(50px) 0
+
+.c-404_message
```
Comment (odincov) [inline web/app/themes/theme/assets/styles/components/404.styl]: double underscore?

**Comment by rslnk** [comment]
@odincov fixed, please check!

**Comment by odincov** [comment]
Approve! Although i think the 404 html structure could be lighter... But this can be optimisations we'll do later.

**Comment by rslnk** [comment]
@odincov agree!

**Pull request merged**